app/src/main/java/com/kamwithk/ankiconnectandroid/Scraper.java

```diff
@@ -16,30 +21,37 @@
 
 // Regular expressions and method source - https://github.com/jamesnicolas/yomichan-forvo-server
 public class Scraper {
+    private final Context context;
     private final String SERVER_HOST = "https://forvo.com";
     private final String AUDIO_HTTP_HOST = "https://audio00.forvo.com";
-    private final String LANGUAGE = "ja";
+
+    public Scraper(Context context) {
+        this.context = context;
+    }
 
     public ArrayList<HashMap<String, String>> scrape(String word, String reading) throws IOException {
-        ArrayList<HashMap<String, String>> audio_sources = scrapeWord(word);
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
+        String forvoLanguage = preferences.getString("forvo_language", "ja");
```

Comment:
It is better to have the default language be a `private final String` constant set to `ja`

Reply:
changed to a const